src/Markdig.Wpf/Renderers/Wpf/Inlines/AutoLinkInlineRenderer.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) 2016-2017 Nicolas Musset. All rights reserved.
+// This file is licensed under the MIT license. 
+// See the LICENSE.md file in the project root for more information.
+
+using System;
+using System.Windows.Documents;
+using Markdig.Annotations;
+using Markdig.Syntax.Inlines;
+using Markdig.Wpf;
+
+namespace Markdig.Renderers.Wpf.Inlines
+{
+    /// <summary>
+    /// A WPF renderer for a <see cref="AutolinkInline"/>.
+    /// </summary>
+    /// <seealso cref="Markdig.Renderers.Wpf.WpfObjectRenderer{Markdig.Syntax.Inlines.AutolinkInline}" />
+    public class AutolinkInlineRenderer : WpfObjectRenderer<AutolinkInline>
+    {
+        /// <inheritdoc/>
+        protected override void Write(WpfRenderer renderer, [NotNull] AutolinkInline link)
+        {            
+            var url = link.Url;
+
+            if (!Uri.IsWellFormedUriString(url, UriKind.RelativeOrAbsolute))
+            {
+                url = "#";
+            }
+
+            var hyperlink = new Hyperlink
+            {
+                Command = Commands.Hyperlink,
```

Comment:
There might be an issue here with hyperlinks and copy/paste (see https://stackoverflow.com/questions/3206258/commandconverter-valid-exception-or-net-bug). We can review that later.